projects/Inventaire/examples/basic.nix

```
@@ -0,0 +1,122 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  # !!! THIS IS INSECURE, DO NOT DO THIS IN PRODUCTION !!!
+  # This is only done like this here to allow easy testing & debugging.
+  # Use some secrets management mechanism to track values like this outside of Nix, so they can't leak into the store
+  couchdbUser = "inventaire";
+  couchdbPassword = "ThisIsNotSecurelyManagedAndImFullyAwareOfThis";
+  couchdbPort = 5984;
+  elasticPort = 9200;
+
+  inventaireServiceDeps = [
+    "couchdb.service"
+    # If using ElasticSearch instead, change this to: "elasticsearch.service"
+    "opensearch.service"
+  ];
+in
+{
+  # !!! THIS IS INSECURE, DO NOT DO THIS IN PRODUCTION !!!
+  # This is just done like this here to allow easy testing & debugging.
+  # Use some secrets management mechanism to get a string of a path at runtime here.
+  environment.etc."inventaire-config-overrides.cjs".text = ''
+    // This is not securely managed, and I'm fully aware of this.
+
+    module.exports = {
```

Needed for the demo (they should probably be configurable, so users can set and change them freely):

```suggestion
    module.exports = {
        hostname: '0.0.0.0',
        port: 3006,
```

We could also link the [config file](https://codeberg.org/inventaire/inventaire/src/commit/bd11c3c6d8dd435caa4e63e67d93b31ace7a754e/config/default.cjs#L61) here, but this might be redundant since it's also in the option docs (but there could be users who won't read those).